### DISCUSSION TITLE
Aleph's vision/roadmap

### DISCUSSION THREAD

**Discussion by phuctm97** [alephjs/aleph.js#72] [General]
Hi Aleph.js team, thanks for creating this framework, I see this is promising to be the next Next.js for me, curious about the vision of the team: When and what will be in v1? How do you guys intend to grow Aleph?

**Comment by shadowtime2000** [comment]
I recently joined the Aleph team so I can't really speak for ije about this, but I believe we are trying to make this comparable feature and speed wise with NextJS, just with the Deno philosophy. v1 will probably take a while (we are roughly a month overdue on v0.3 I believe). Sorry, I just don't know how to exactly answer your last question here. I think this is a discussion worth having so I am going to pin this.

**Reply by phuctm97** [reply]
Got it, thank you @shadowtime2000. Meanwhile, what should I do if I want to contribute?

**Reply by shadowtime2000** [reply]
@phuctm97 Check out the issues and help around with them, open some issues because that helps too, you can also check out the other AlephJS projects [in the GH org](https://github.com/alephjs).

**Comment by ije** [comment]
hi @phuctm97, i think the **v1** may take one or two years since the Deno is not stable currently. I'm working on **v0.3** but i was blocked because the bundler of swc has some issues that is deno's internal bunlder. please check #28

**Reply by ije** [reply]
and please check [Deno's Q1 2021 roadmap](https://github.com/denoland/deno/issues/8824), **v0.3** will work on Deno 1.7+.

**Reply by phuctm97** [reply]
Hey, @ije one year actually sounds like a good plan. Will try to help as much as possible.

**Reply by ije** [reply]
@phuctm97 thanks!
